hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -50,7 +50,32 @@ There are separate translations controls for instruction and data.
 Each control contains two programmable regions (2 for instruction and 2 for data).
 If a transaction matches multiple regions, the lowest indexed region has priority.
 
-For details on how to program the related registers, please see {{< regref "IBUS_ADDR_MATCHING0" >}} and {{< regref "IBUS_REMAP_ADDR0" >}}
+For details on how to program the related registers, please see {{< regref "IBUS_ADDR_MATCHING_0" >}} and {{< regref "IBUS_REMAP_ADDR_0" >}}.
+
+## Random Number Generation
+
+The wrapper has a connection to the [Entropy Distribution Network (EDN)]({{< relref "hw/ip/edn/doc" >}}) with a register based interface.
+The {{< regref "RND_DATA" >}} register provides 32-bits directly from the EDN.
+{{< regref "RND_STATUS.RND_DATA_VALID" >}} indicates if the data in {{< regref "RND_DATA" >}} is valid or not.
+A polling style interface is used to get new random data.
+Any read to {{< regref "RND_DATA" >}} when it is valid invalidates the data and triggers an EDN request for new data.
+Software should poll {{< regref "RND_STATUS.RND_DATA_VALID" >}} until it is valid and then read from {{< regref "RND_DATA" >}} to get the new random data.
+The {{< regref "RND_STATUS.RND_DATA_PENDING" >}} bit indicates whether an EDN request for new data is pending.
+Either {{< regref "RND_STATUS.RND_DATA_VALID" >}} will be set or {{< regref "RND_STATUS.RND_DATA_PENDING" >}} will be set.
+They will never both be clear or both be set at once.
```

Comment:
I think I'm back to not understanding. This text says that `RND_DATA_VALID = ! RND_DATA_PENDING`. That doesn't seem very useful to me! Wasn't the point that having both these flags simplified the logic at startup (when both were clear)? If we go with the design described in the next paragraph, I don't see the point in the RND_DATA_PENDING bit.

Reply:
>  I don't see the point in the RND_DATA_PENDING bit.

You're quite right. With the current design it's not particularly useful. I'll drop that bit and add to the documentation to explain that either the data is pending or it is valid, there is no other state (so in particular we're pending at reset but need to enable EDN before the request is dealt with).